notebooks/Stable-Diffusion-local-Windows.ipynb

```diff
@@ -0,0 +1,222 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Easy-peasy Windows install"
```

Comment:
Is this bit related to the Model handling? This seems to be outside the scope of PR?

Reply:
(comment pertains to the entire Windows.ipynb)

Reply:
I've removed this file from the PR.